src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,36 @@
-const getEmployerRole = (employeeName, employees) => {};
+/* const getEmployerRole = (employeeName, employees) => {
+  const search = employees.filter((y) => y.name === employeeName);
+  return String(search.map((x) => x.role)); 
+}; */
```

Comment:
Unused commented out functions should always be removed, they just add confusion to the flow of the file and dont serve any useful purpose